src/pexprs-toArgumentNameList.js

```diff
@@ -56,7 +56,13 @@ pexprs.Terminal.prototype.toArgumentNameList = function(firstArgIndex) {
 };
 
 pexprs.Range.prototype.toArgumentNameList = function(firstArgIndex) {
-  return [this.from + '_to_' + this.to];
+  var argName = this.from + '_to_' + this.to;
+  // If the `argName` doesn't start with an valid character, i.e. alphabet or `_`,
+  // then prepending a `_` to it.
+  if (!/[_$a-zA-Z]/.test(argName[0])) {
+    argName = '_' + argName;
```

Comment:
This could still result in invalid identifiers, right? E.g. a range `"!".."@"`. I think the only time appending an underscore will help is if argName[0] is a digit. Also, this will incorrectly append an underscore for things that begin with non-ASCII characters (e.g. `ä`).

What you might want to do is write a utility function that will check if a given string is a valid JS identifier, and then in this method, first try `this.from + '_to_' + this.to`, then try appending `_`,  then fall back to a positional argument name (like `Terminal` does).